Tests/LibWeb/Text/expected/Streams/QueuingStrategy-high-water-mark.txt

```diff
@@ -0,0 +1,20 @@
+======================================
+CountQueuingStrategy
+======================================
+'{"highWaterMark":2}' => 2
+'{"highWaterMark":-1}' => -1
+{"badKey":-1}' => Exception raised of TypeError
+{}' => Exception raised of TypeError
```

Comment:
I am not too familiar with our text tests, but why are these missing the single quote at the beginning?

```suggestion
'{"badKey":-1}' => Exception raised of TypeError
'{}' => Exception raised of TypeError
```

Reply:
:facepalm: I missed the ' in the `println` in the test. Fixed, thanks!